samples: openthread: Fix rcp_hci variant of coprocessor sample (#30090)
nrfconnect/sdk-nrf: samples/openthread/coprocessor/README.rst

         west build -b *board_target* -p -- -DEXTRA_CONF_FILE=extra_conf/rcp_hci.conf -DEXTRA_DTC_OVERLAY_FILE=extra_conf/rcp_hci.overlay

> To attach Bluetooth HCI on your host machine, you can use the following command:

Reviewer: I think it'll make more sense to move this info to the top (removed from the line 194)
```suggestion
To attach Bluetooth HCI on your host machine, run the command for your operating system, replacing ``serial_port`` with the serial port of the device you want to connect to:
```

.. tabs::

   .. tab:: Linux

      .. parsed-literal::
         :class: highlight

         sudo btattach -B *serial_port* -S 1000000 -P h4 -N

>       for example:

Reviewer: ```suggestion
      For example:
```

      .. parsed-literal::
         :class: highlight

         sudo btattach -B /dev/ttyACM0 -S 1000000 -P h4 -N

   .. tab:: Windows

      .. parsed-literal::
         :class: highlight

         gatt_counter.exe -u *serial_port* --baud 1000000

>       for example:

Reviewer: ```suggestion
      For example:
```

      .. parsed-literal::
         :class: highlight

         gatt_counter.exe -u COM1 --baud 1000000

> Where ``serial_port`` is the serial port of the device you want to connect to.

Reviewer: ```suggestion
```
